modules/temp-deck/temp-deck-arduino/temp-deck-arduino.ino

```diff
@@ -134,6 +138,7 @@ PID myPID(&CURRENT_TEMPERATURE, &TEMPERATURE_SWING, &TARGET_TEMPERATURE, DOWN_PI
 
 String device_serial = "";  // leave empty, this value is read from eeprom during setup()
```

Comment:
We should consider making these local to `setup` if possible, not sure how much I like this combination of the free store and the heap

Reply:
We use `device_serial` & `device_model` in device_info gcode response as well. And it's very much possible we might use it in other contexts in the future esp with the Gen2 tempdecks if they have any changes that might need us to add conditions based on the device_model. So, I think keeping it global makes sense